### DISCUSSION TITLE
Condicionar apoio da APyB a eventos que façam campanha de filiação de novos membros

### DISCUSSION THREAD

**Discussion by ericof** [apyb/comunidade#182] [Geral]
Eu gostaria de sugerir que a APyB condicione o apoio aos eventos regionais, locais e grupos de usuário a realização de uma campanha para a filiação de novas pessoas a APyB.

Essa campanha pode ser liderada pela diretoria de marketing da APyB -- na PythonCerrado2024 a APyB imprimiu um folheto divulgando suas ações e com link para o linktree, mas sem uma ação específica para nós associarmos.

**Comment by belisarenata** [comment]
Pequena correção: os folhetos estavam disponíveis na PythonCerrado2024.

**Reply by ericof** [reply]
Corrigido!

**Comment by tiidadavena** [comment]
Follow-up: adicionamos a sugestão no Termo de Compromisso para pessoas Big Kahunas.
